packages/botonic-react/src/components/multichannel/multichannel-text.jsx

```diff
@@ -77,11 +81,14 @@ export const MultichannelText = props => {
       const asText =
         type === buttonTypes.POSTBACK ? postbackButtonsAsText : true
       const newline =
-        multichannelContext.messageSeparator == null &&
+        multichannelContext.messageSeparator === null &&
```

Comment:
Observation/Change: Adding the `===`, if the `messageSeparator` is `undefined` the condition will return `false` and I think this is not the wanted behaviour. 
I would leave it as == or check both values `null` and `undefined`.

Reply:
Yes you are right. I'm changing the logic with this triple equal...